README.md

```diff
@@ -21,3 +21,10 @@ If you update or add a new language file, please also update the below table wit
 ### Contributing
 
 Everybody is welcome to contribute! You can help by sending us a pull request with your translation.
+Please keep the naming conventions:
+
+- localisation file `X.json`
+- whats new: `whatsnew_X.md`
+
+where `X` is the two letter language abbreviation.
+Please also put your files into proper version folder.
```

Comment:
there is a "### List of available translations" section in the README. Can this now be deleted?

Reply:
technically it can but maybe it is worth keeping?